commit ed7577cb855337d9a93e66fa0eb73eb604fe8f04 (refs/changes/89/2389/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-08-14T21:50:24+00:00 (2 years, 2 months ago)
    
    OS-6295 t4nex driver missing pciex1425,6405
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index f9d9080e..eda1c7b9 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -1162,7 +1162,7 @@ t4nex "pciex1425,6401"
 t4nex "pciex1425,6402"
 t4nex "pciex1425,6403"
 t4nex "pciex1425,6404"
-t4nex "pciex1425,6404"
+t4nex "pciex1425,6405"
 t4nex "pciex1425,6406"
 t4nex "pciex1425,6407"
 t4nex "pciex1425,6408"
